dapps/src/apps/fs.rs

```diff
@@ -97,9 +99,22 @@ fn read_manifest(name: &str, mut path: PathBuf) -> EndpointInfo {
 		})
 }
 
-pub fn local_endpoints(dapps_path: String, signer_address: Option<(String, u16)>) -> Endpoints {
+pub fn local_endpoint<P: AsRef<Path>>(path: P, signer_address: Option<(String, u16)>) -> Option<(String, Box<LocalPageEndpoint>)> {
```

Comment:
would be nice to have short doc comments for non-trivial functions, even if they're non-visible outside their crates.